### PR TITLE
chore: fix syntax error in CODEOWNERS file

### PR DESCRIPTION
**Pull request by FeynmanZhou**
fix syntax problems in CODEOWNERS and MAINTAINERS

### CODE REVIEW

**Review by iamsamirzon — DISMISSED** [review]

**Review by toddysm — DISMISSED** [review]
Little nit but for aesthetic purposes we should be consistent in our use of casing :) *org-level* should be *Org-Level* for consistency.

**Review by yizha1 — DISMISSED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
I fixed the git signing problem due to the branch policy changes in https://github.com/notaryproject/notaryproject/issues/226. Please review and re-approve this PR. Thanks

**Review by iamsamirzon — DISMISSED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,2 @@
...
-# Repo-Level Owners (in alphabetical order)
-# Note: This is only for the notaryproject/notaryproject repo
-* @gokarnm
-* @priteshbandi
-* @shizhMSFT
-* @toddysm
-* @vaninrao10
-* @yizha1
+# Org-Level and repo-Level Owners (in alphabetical order). Please note this is only for the notaryproject/notaryproject repo
+* @gokarnm @justincormack @niazfk @priteshbandi @stevelasker @shizhMSFT @toddysm @vaninrao10 @yizha1
```
Comment (toddysm) [inline CODEOWNERS]: Please add an empty new line at the end per convention

File: MAINTAINERS [hunk 2]
```diff
@@ -11,4 +11,4 @@ Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
 Shiwei Zhang <shizh@microsoft.com> (@shizhMSFT)
 Toddy Mladenov <toddysm@gmail.com> (@toddysm)
 Vani Rao <vaninrao@amazon.com> (@vaninrao10)
-Yi Zha <yizha1@microsoft.com> @yizha1
+Yi Zha <yizha1@microsoft.com> @yizha1
```
Comment (toddysm) [inline MAINTAINERS]: Please add an empty new line at the end per convention

**Review by toddysm — CHANGES_REQUESTED** [review]

**Review by vaninrao10 — DISMISSED** [review]
@toddysm good catch on the empty line. @FeynmanZhou please take care. 
Rest LGTM

**Comment by yizha1** [comment]
@FeynmanZhou  Is this PR still valid? Maybe we can close it.

**Comment by toddysm** [comment]
@yizha1 We need to add empty lines at the end of the file. This change was requested some time back. Once this is done, we can metge

File: CODEOWNERS (hunk 1 above)
Comment (FeynmanZhou) [reply CODEOWNERS]: @toddysm Sorry for the late response. What's the reason for adding an empty new line? It seems this syntax rule is not defined and required in https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

File: MAINTAINERS (hunk 2 above)
Comment (FeynmanZhou) [reply MAINTAINERS]: Updated.

**Comment by FeynmanZhou** [comment]
Hi @toddysm @iamsamirzon @vaninrao10 @yizha1 @priteshbandi ,

I resolved @toddysm 's comments. Could you pls review and approve it again? Thanks

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by toddysm — APPROVED** [review]

**Review by justincormack — APPROVED** [review]

**Comment by justincormack** [comment]
Maybe could squash some of this into fewer commits...

**Pull request merged**